### DISCUSSION TITLE
Recovery failure

### DISCUSSION THREAD

**Discussion by nathanael2022** [iotaledger/firefly#2320] [Help & Support]
Hi, I recovered my Fireflywallet and I can't see my balance.. 

The adress is also different than the one I first transferred to, when I look that adress up on Iota explorer I can see the balance..
Anybody knows what I've done wrong or what has happened?

**Comment by MiKeF05** [comment]
Hello,
so in the old profile you only used the first address? And now you restored your profile with your 24 words and the address is different? If the first address is now different then most likely you use other 24 words. Because the address is derived from the 24 words. Could it be that you use other 24 words?
